ssn_trace_test.go

```diff
@@ -0,0 +1,25 @@
+package checkr
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const driverpackage = "driver_pro"
```

Comment:
This would be camelcase, `driverPro`. Actually, it would be good if you created a type for this.

Reply:
i changed the camel case. when you say type do you mean iota and creating a struct for it. later use their string representation to pass for package during report generation ? Like the http.StatusOK stuff ?

Reply:
@chandradeepak yeah, something like that. Because in `detektiv` we will need a way to choose the package we want to run. You can see `gblcodes` how I did it.

Reply:
@magicalbanana fixed this  with codes